### ISSUE TITLE
Some RLE encoded segmentations cannot be read with readFile

### ISSUE THREAD

**Issue by emelalkim** [dcmjs-org/dcmjs#78]
When you create a segmentation, by default it compresses with RLE encoding. But readFile doesn't recognize RLE Encoded transfer syntax and normalizes to EXPLICIT_LITTLE_ENDIAN. This causes "Range Error: Source is too large" error on some of the segmentations, not all of them. I uploaded a [sample segmentation](https://drive.google.com/open?id=1YQIBARbvUwy9Rp0ekbDs04RS0VHowdZN) that causes this error to google drive. @muakdogan

**Comment by pieper** [comment]
Yes, makes sense.  Do you want to propose a fix?

**Comment by emelalkim** [comment]
I tried removing the normalization but it caused other issues. The reading code needs to be changed to handle RLE encoding. We decided not use RLE encoding right now, I'll try to take a look when I have time.

**Comment by pieper** [comment]
Yes, that is good.  There's been a lot of discussion about how best to handle RLE and bitpacked seg objects most efficiently.
